### DISCUSSION TITLE
Uk prices are available

### DISCUSSION THREAD

**Discussion by AlexandreCouderc** [electricitymaps/electricitymaps-contrib#3706] [Q&A] [answered]
Apparently UK prices are back as seen here: [https://www.rte-france.com/eco2mix/les-donnees-de-marche](https://www.rte-france.com/eco2mix/les-donnees-de-marche)
Don't know where the data comes from.

**Comment by madsnedergaard** [comment, marked as answer]
Fixed by you in #3709 👏
